docs/UserGuide.md

```diff
@@ -209,6 +209,24 @@ Restrictions:
 Examples:
 * `ecreate n/food donation r/chef r/packer d/23-9-2023 1500 dsc/help food distribution m/50 potatoes b/50` creates an event with name `food donation`, roles needed `chef` and `packer`, event date `23rd September 2023, 3pm`, description `help food distribution`, materials needed `50 potatoes` and budget `$50`
 
+### List all events : `elist` [coming soon]
```

Comment:
Can change `List` to `Listing` to maintain consistency with other features.

Reply:
i agree

Reply:
oki